packages/commonwealth/package.json

```diff
@@ -16,6 +16,7 @@
     "create-migration": "npx sequelize migration:generate --name",
     "datadog-db-setup": "chmod u+x scripts/setup-datadog-postgres.sh && ./scripts/setup-datadog-postgres.sh",
     "db-all": "yarn reset-db && yarn load-db && yarn migrate-db",
+    "db-doc": "chmod u+x scripts/gen-mermaid-erd.sh && ./scripts/gen-mermaid-erd.sh > ../../knowledge_base/wiki/ERD.md",
```

Comment:
If we could get a few lines of documentation for this script, that'd be awesome. See `Package-Scripts.md` in our knowledge base.

Reply:
Sorry, missed that step!